src/generators/ast.ts

```diff
@@ -198,10 +204,97 @@ function copyAdvancedKeywords(from: JSONSchema7, to: Schema): void {
   }
 }
 
+type ResolvedTypes = {
+  [key: string]: Schema;
+};
+
+let definedTypes: ResolvedTypes = {};
+
+export interface CustomType {
+  name: string;
+  schema: Schema;
+}
+
+export const customTypesByEvent: Record<string, CustomType[]> = {};
```

Comment:
Where are we using this `exported` `const` ?

Reply:
we are using this in JS SDK generator